app/views/policies_mailer/policy_update_announcement.html.erb

```diff
@@ -0,0 +1,21 @@
+
+<% @title = t(".title") %>
+<% @sub_title = t(".subtitle", handle: @user.handle) %>
+
+<!-- Body -->
+<table width="100%" border="0" cellspacing="0" cellpadding="0" bgcolor="#ffffff">
+  <tr>
+    <td class="content-spacing" style="font-size:0pt; line-height:0pt; text-align:left" width="20"></td>
+    <td>
+      <table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="35" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
+
+      <div class="h3-1-center" style="color:#1e1e1e; font-family:Georgia, serif; min-width:auto !important; font-size:20px; line-height:26px;">
+        <p>We're reaching out to inform you that RubyGems.org is adopting a <a href="https://rubygems.org/policies/terms-of-service">Terms of Service</a>, <a href="https://rubygems.org/policies/privacy">Privacy Notice</a>, <a href="https://rubygems.org/policies/acceptable-use">Acceptable Use Policy</a>, and <a href="https://rubygems.org/policies/copyright">Copyright Policy</a>. We announced the review period for these policies in our <a href="https://blog.rubygems.org/2025/03/20/introducing-new-policies.html">recent blog post</a>. We encourage all users to review these policies. We are seeking feedback on the policies for the next 30 days. Please reply by email to legal@rubycentral.org or leave a message in the <a href="https://rubycentralcommunity.slack.com/archives/C08J92C73N1">#oss-program-ruby-central channel of the Ruby Central Community Slack</a>.</p>
```

Comment:
I know it is too late now, but I'm wondering @mghaught, what's the reason to use Ruby Central Community Slack in here instead of Bundler/RubyGems(.org) one?